master/buildbot/test/unit/test_status_github.py

```diff
@@ -170,11 +170,28 @@ def test_startService(self):
 
         self.status._status.subscribe.assert_called_with(self.status)
 
+    def test_stopService(self):
+        """
+        When stopped it will unsubscribe from all statuses.
+        """
+        builderstatus = FakeBuilderStatus()
+
+        result = self.status.builderAdded('builder-name', builderstatus)
```

Comment:
we really need to port this. if you are volunteer that would be very cool
see the status, so that you can find example of ported code:
http://docs.buildbot.net/latest/manual/cfg-reporters.html#githubstatus-not-migrated